main.go

```diff
@@ -67,3 +69,37 @@ func main() {
 	fmt.Printf("Documents scanned: %v | Lines scanned: %v | Translations needed: %v\n", docs, lines, inNeed)
 	os.Exit(0)
 }
+
+// getOriginalDir gets the default Steam installation directory.
+// Returns empty string if directory is not found.
+func getOriginalDir() string {
+	rimWorldDir := ""
+
+	// https://support.steampowered.com/kb_article.php?ref=7710-tdlc-0426
+	if runtime.GOOS == "windows" && runtime.GOARCH == "386" {
+		rimWorldDir = `C:\Program Files\Steam\steamapps\common\RimWorld`
+	} else if runtime.GOOS == "windows" && runtime.GOARCH == "amd64" {
+		rimWorldDir = `C:\Program Files (x86)\Steam\steamapps\common\RimWorld`
+	} else if runtime.GOOS == "darwin" {
+		// https://gaming.stackexchange.com/a/219537
+		currentUser, err := user.Current()
+		if err != nil {
+			return ""
+		}
+
+		// safe to use *nix path sep since we are in the darwin runtime
+		rimWorldDir = filepath.Join(currentUser.HomeDir,
+			`/Library/Application Support/Steam/steamapps/common/RimWorld/RimWorldMac.app`)
```

Comment:
you can use this mac path (from readme as an example here :D) `/Users/arthur/Library/Application Support/Steam/steamapps/common/RimWorld/RimWorldMac.app/Mods/Core/Languages/English`. You'll just need to update it to the current user's name with `currentUser.Name`.

Reply:
I believe I have that implemented. currentUser.HomeDir gives the absolute path to your home dir. Then I just join the Steam path to the given directory.

Reply:
Sorry, my bad, did not get to see the follow up